.travis.yml

```diff
@@ -29,7 +29,10 @@ before_install:
   - opam update
   - opam switch "$OCAML_VERSION"
   - eval $(opam env)
-  - opam install dune menhir bindlib.5.0.0 timed.1.0 earley.2.0.0 yojson cmdliner ppx_inline_test $EXTRA_OPAM
+  - opam install dune menhir bindlib.5.0.0 timed.1.0 earley.2.0.0 yojson cmdliner ppx_inline_test why3 $EXTRA_OPAM
+  # For why3 provers [altergo]
+  - opam install alt-ergo
```

Comment:
Only use on `opam install` command, you can add this to the list above (this is more efficient and prevents conflicts in version numbers).